server/spawn-vehicle.lua

```diff
@@ -5,19 +5,17 @@
 ---@return boolean
 local function checkHasAccessToVehicle(source, garageName, garageType, vehicleId)
     local player = exports.qbx_core:GetPlayer(source)
+    local result = nil
```

Comment:
```suggestion
    local result
```
This is unused, see linter